dists/win32/ScummVM.iss

```diff
@@ -1,251 +1,287 @@
-begin
-  { Collect current instance parameters }
-  for I := 1 to ParamCount do
-  begin
-    S := ParamStr(I);
-    { Unique log file name for the elevated instance }
-    if CompareText(Copy(S, 1, 5), '/LOG=') = 0 then
-    begin
-      S := S + '-elevated';
-    end;
-    { Do not pass our /SL5 switch }
-    if CompareText(Copy(S, 1, 5), '/SL5=') <> 0 then
-    begin
-      Params := Params + AddQuotes(S) + ' ';
-    end;
-  end;
-
-  { ... and add selected language }
-  Params := Params + '/LANG=' + ActiveLanguage;
-
-  Log(Format('Elevating setup with parameters [%s]', [Params]));
-  RetVal := ShellExecute(0, 'runas', ExpandConstant('{srcexe}'), Params, '', SW_SHOW);
-  Log(Format('Running elevated setup returned [%d]', [RetVal]));
-  Result := (RetVal > 32);
-  { if elevated executing of this setup succeeded, then... }
-  if Result then
-  begin
-    Log('Elevation succeeded');
-    { exit this non-elevated setup instance }
-    ExitProcess(0);
-  end
-    else
-  begin
-    Log(Format('Elevation failed [%s]', [SysErrorMessage(RetVal)]));
-  end;
-end;
-
-procedure InitializeWizard;
-var
-  S: string;
-  Upgrade: Boolean;
-begin
-  Upgrade :=
-    RegQueryStringValue(HKLM, '{#InnoSetupReg}', '{#InnoSetupAppPathReg}', S) or
-    RegQueryStringValue(HKCU, '{#InnoSetupReg}', '{#InnoSetupAppPathReg}', S);
-
-  { elevate }
-
-  if not IsWinVista then
-  begin
-    Log(Format('This version of Windows [%x] does not support elevation', [
-      GetWindowsVersion]));
-  end
-    else
-  if IsElevated then
-  begin
-    Log('Running elevated');
-  end
-    else
-  begin
-    Log('Running non-elevated');
-    if Upgrade then
-    begin
-      if not HaveWriteAccessToApp then
-      begin
-        Elevate;
-      end;
-    end
-      else
-    begin
-      if not Elevate then
-      begin
-        WizardForm.DirEdit.Text := ExpandConstant('{localappdata}\{#AppName}');
-        Log(Format('Falling back to local application user folder [%s]', [
-          WizardForm.DirEdit.Text]));
-      end;
-    end;
-  end;
-end;
+#define AppName "ScummVM"
+#define FileVersion=GetFileVersion('scummvm.exe') 
+#define ProductVersion=GetStringFileInfo('scummvm.exe', 'ProductVersion')
+#define Copyright=GetStringFileInfo('scummvm.exe', 'LegalCopyright')
+
+[Setup]
+AppCopyright={#Copyright}
+AppName={#AppName}
+AppVersion={#ProductVersion}
+VersionInfoVersion={#FileVersion}
+AppVerName={#AppName} {#ProductVersion}
+AppPublisher=The ScummVM Team
+AppPublisherURL=http://www.scummvm.org/
+AppSupportURL=http://www.scummvm.org/
+AppUpdatesURL=http://www.scummvm.org/
+DefaultDirName={pf}\{#AppName}
+DefaultGroupName={#AppName}
+AllowNoIcons=true
+AlwaysUsePersonalGroup=false
+EnableDirDoesntExistWarning=false
+Compression=lzma2/ultra
+OutputDir=C:\{#AppName}
+OutputBaseFilename=scummvm-win32
+DisableStartupPrompt=true
+AppendDefaultDirName=false
+SolidCompression=true
+DirExistsWarning=no
+SetupIconFile=graphics\scummvm-install.ico
UninstallDisplayIcon={app}\scummvm.exe
```

Comment:
Shouldn't there be a new line between these two?

Reply:
Thanks, seems that Inno Studio screwed up the encoding - again. Will solve this ASAP obviously.

Reply:
After investigation, I can say that the file itself is correct, this seems to be an rendering/displaying error in Github.